tests/test_commands.py

```diff
@@ -42,6 +42,7 @@ def is_true(option: str) -> bool:
 serve_command = parametrized_fixture(serve=['serve'],
                                      freeze_serve=['freeze', '--serve'])
 port = parametrized_fixture(8001, 8080)
+host = parametrized_fixture('localhost', '127.0.0.1', '0.0.0.0')
```

Comment:
What about this as well?

```suggestion
host = parametrized_fixture('localhost', '127.0.0.1', '::1', '0.0.0.0')
```

(Assuming it work on the CI.)

Reply:
Looks like Travis CI doesn't support IPv6 very well.
Also, `::1` becomes `[::1]` in URLs, which makes the tests less clear, and I don't think really tests much in Elsa.
So, I'll pass on this.